car_simulator/source/server.cpp

```diff
@@ -38,12 +41,148 @@ class Server : public std::enable_shared_from_this<Server> {
     }
   }
 
+  inline void reply_and_log(const std::string & response, const std::string & log_message, boost::system::error_code & error_write_back)
+  {
+    socket.write_some(boost::asio::buffer(response.c_str(), response.size()), error_write_back);
+    logger->log(log_message);
+  }
+
+  void prepare_message_for_parsing(const std::size_t bytes_transferred)
```

Comment:
We do not need  `bytes_transferred` anymore.

Reply:
Fixed